### PR TITLE
Patch variable name error

### PR DESCRIPTION
**Pull request by dstallenberg**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
+## [0.2.2] - 2023-01-25
```
Comment (mitchellolsthoorn) [inline CHANGELOG.md:22]: This should probably be unreleased as this version is not released yet

File: src/SolidityLauncher.ts [hunk 2]
```diff
@@ -423,6 +427,14 @@ export class SolidityLauncher {
     console.log = old;
     await this.api.onTestsComplete(this.config);
 
+    const mochaRunner: Runner = this.truffle.test.mochaRunner;
```
Comment (mitchellolsthoorn) [inline src/SolidityLauncher.ts:430]: What does this have to do with the generation of variable names bug?

**Review by mitchellolsthoorn — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dstallenberg) [reply CHANGELOG.md:22]: Fixed.

File: src/SolidityLauncher.ts (hunk 2 above)
Comment (dstallenberg) [reply src/SolidityLauncher.ts:430]: Well while trying to debug it was not printing that a testcase had failed.
So I added this. I think its good to keep it in here.

File: CHANGELOG.md (hunk 1 above)
Comment (mitchellolsthoorn) [reply CHANGELOG.md:22]: I don't see the fix in the PR

Comment (dstallenberg) [reply CHANGELOG.md]: Forgot to push, my bad.

**Review by mitchellolsthoorn — APPROVED** [review]

**Comment by mitchellolsthoorn** [comment]
Don't we now have the random number generator with support for alphabets? This would make the following change somewhat unnecessary, right?

**Comment by dstallenberg** [comment]
> Don't we now have the random number generator with support for alphabets? This would make the following change somewhat unnecessary, right?

In the end this PR is more about the types that where in the names.

For example if the type of an argument was an array the name of the argument would include "[]" which would crash the test case.

So this patch is still required.
